src/NuGet.Status/Controllers/AuthorizedController.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System.Web.Mvc;
+using Microsoft.WindowsAzure.Storage.Table;
+using NuGet.Services.Status.Table.Manual;
+using NuGet.Status.Helpers;
+using NuGet.Status.Models;
+using NuGet.Status.Utilities;
+
+namespace NuGet.Status.Controllers
+{
+    public class AuthorizedController : AppController
```

Comment:
>AuthorizedController [](start = 17, length = 20)

The name of the controller doesn't match what it does..

Reply:
You split these just because of the auth? You really shouldn't need two controllers to make auth work, re-merge them.